Bottom: 3c8daa01447616bb3b6e3c7d496ae8e209bfbbfa
Top:    d2c2c113002084b9f238822a8dd7fbf4e1e454f0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-01-09 17:30:16 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
